packages/common/src/store/ui/modals/withdraw-usdc-modal/index.ts

```diff
@@ -2,7 +2,9 @@ import { createModal } from '../createModal'
 
 export enum WithdrawUSDCModalPages {
   ENTER_TRANSFER_DETAILS = 'enter_transfer_details',
+  COINFLOW_TRANSFER = 'coinflow_transfer',
   CONFIRM_TRANSFER_DETAILS = 'confirm_transfer_details',
+  PREPARE_TRANSFER = 'prepare_transfer',
```

Comment:
is this state specific to coinflow? should we prepend with `COINFLOW_`?

Reply:
Yes and no. It's necessary to move it to a "real" wallet before attempting to withdraw it through a prebuilt third-party flow. But the only one we have right now is coinflow. I was trying to avoid being overly specific with the naming. Especially since this step in the modal literally just says "Preparing Transfer" with no reference to the vendor.